src/model_io/urdf/src/URDFModelExport.cpp

```diff
@@ -106,10 +106,120 @@ bool exportInertial(const SpatialInertia &inertia, xmlNodePtr parent_element)
     return ok;
 }
 
+
+
+/*
+ * Add a <visual> or <collision> URDF element to the specified parent element for the specified solid shapes.
+ *
+ * If parent_element contains a pointer to a tag <parent_element></parent_element>, this function modifies it to be something like:
+ * <parent_element>
+ *   <visual>
+ *     <origin xyz="0 0 0" rpy="0 0 0" />
+ *     <geometry>
+ *       <box size="1 1 1" />
+ *     </geometry>
+ *     <material name="Cyan">
+ *       <color rgba="0 1.0 1.0 1.0"/>
+ *     </material>
+ *   </visual>
+ * <parent_element/>
+ * where the actual values of the added element and attributes depend on the input solidShape.
+ */
+enum exportSolidShapeType {
+    VISUAL,
+    COLLISION
+};
+bool exportSolidShape(const SolidShape* solidShape, exportSolidShapeType type, xmlNodePtr parent_element)
+{
+    bool ok=true;
+    std::string element_name;
+    if (type == VISUAL) {
+        element_name = "visual";
+    } else {
+        // assert(type == COLLISION)
+        element_name = "collision";
+    }
+
+    xmlNodePtr root_shape_xml = xmlNewChild(parent_element, NULL, BAD_CAST element_name.c_str(), NULL);
+    std::string shapeName = solidShape->name;
```

Comment:
this line can be brought inside the if block, since it's not used anywhere else.